python/tk_multi_workfiles/browser_form.py

```diff
@@ -1,11 +1,11 @@
 # Copyright (c) 2015 Shotgun Software Inc.
-# 
```

Comment:
I've limited whitespaces fixes to lines with comments or empty lines as to not confuse git blame.